docs/scale/02-deploy-infra.md

```diff
@@ -0,0 +1,218 @@
+
+2. Choose **Start VPC Wizard**.
+
+3. Make sure **VPC with a Single Public Subnet** is selected.
+
+    ![](../images/vpc-create-1.png)
+
+4. Click **Select**.
+
+    The browser displays the **Step 2: VPC with a Single Public Subnet** dialog.
+
+5. Complete the dialog as follows:
+
+    ![](../images/vpc-create-2.png)
+
+6. Click **Create VPC**.
+
+    AWS works to build the VPC and then presents you with the **VPC Successfully
+    Created** page.
+
+7. Click **OK**.
+
+8. Choose **Subnets** from the **VPC Dashboard** menu.
+
+9. Locate your `publicSwarm` subnet.
+
+10. Choose **Subnet Actions > Modify Auto-Assign Public IP**.
+
+    ![](../images/vpc-create-3.png)
+
+11. Select **Enable auto-assign Public IP** and click **Save**.
+
+In the next step, you configure the remaining AWS settings by using a
+CloudFormation template.
+
+## Step 2. Build the network stack
+
+In this step, you use CloudFormation template to build a stack on AWS. Before
+you begin, make sure you have the prerequisites:
+
+-  access to a private key pair associated with your AWS account.
+-  a clone or download of the <a
+href="https://github.com/docker/swarm-microservice-demo-v1" target="_blank">the
+example code</a> on your local machine.
+
+Then, do the following:
+
+1. Go to the AWS console and choose **CloudFormation**.
+
+    ![](../images/vpc-create-4.png)
+
+2. Click **Create Stack**.
+
+3. Under **Choose a template** click the **Choose file** button.
+
+4. Browse to the download sample code and choose the  the `swarm-microservice-demo-v1/AWS/cloudformation.json` CloudFormation template.
+
+    ![](../images/vpc-create-5.png)
+
+5. Click **Next**.
+
+    The system pre-populates most of the  **Specify Details** dialog from the template.
+
+6. Name the stack `VotingAppStack`.
+
+    You can name the stack something else if you want just make sure it is meaningful.
+
+7. Select your key pair from the **KeyName**  dropdown.
+
+8. Select the `publicSwarm` for the **Subnetid** dropdown menu.
+
+9. Select `swarm-scale` from the **Vpcid** dropdown menu.
+
+10. Click **Next** twice to reach the **Review** page.
+
+11. Check the values.
+
+    The **Template URL**,**SubnetId** and **VpcId** are always unique, so yours
+    will not match, but otherwise you should see the following:     
+
+    ![](../images/vpc-create-6.png)
+
+12. Click **Create**.    
+
+    AWS displays the progress of your stack being created
+
+    ![](../images/create-stack.gif)
+
+## Step 3. Check your deployment
+
+When completed, the CloudFormation populates your VPC with six EC2 instances.
+
+| Instance     | Size      | Private IP Address |
+|--------------|-----------|--------------------|
+| `frontend01` | t2.micro  | 192.168.33.20      |
+| `frontend02` | t2.micro  | 192.168.33.21      |
+| `interlock`  | t2.micro  | 192.168.33.12      |
+| `manager`    | t2.micro  | 192.168.33.11      |
+| `store`      | m3.medium | 192.168.33.250     |
+| `worker01`   | t2.micro  | 192.168.33.200     |
```

Comment:
I'd suggest adding these to /etc/hosts file. And use `name` to reference each machine instead of IP address.